docs-source/content/quickstart/create-domain.md

```diff
@@ -17,58 +22,22 @@ weight: 6
     with the value you provided.  For example, the command above would create a secret named
     `sample-domain1-weblogic-credentials`.
 
-1.	Create a new image with a domain home by running the [`create-domain`](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script.
-Follow the directions in the [sample]({{< relref "/samples/simple/domains/domain-home-in-image/_index.md" >}}),
-including:
-
-    * Copying the sample `kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml` file and updating your copy with the `domainUID` (`sample-domain1`),
-domain namespace (`sample-domain1-ns`), and the `domainHomeImageBase` (`container-registry.oracle.com/middleware/weblogic:12.2.1.3`).
-
-    * Setting `weblogicCredentialsSecretName` to the name of the secret containing the WebLogic credentials, in this case, `sample-domain1-weblogic-credentials`.
-
-    * Leaving the `image` empty unless you need to tag the new image that the script builds to a different name.
-{{% notice note %}}
-If you set the `domainHomeImageBuildPath` property to `./docker-images/OracleWebLogic/samples/12213-domain-home-in-image-wdt`, make sure that your `JAVA_HOME` is set to a Java JDK version 1.8 or later.
-{{% /notice %}}
+1.	Create a new image with a domain home by running the [create-domain](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script. First, copy the sample [create-domain-inputs.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml) file and update your copy with:  
+       * `domainUID`: `sample-domain1`
+       * `image`: Leave empty unless you need to tag the new image that the script builds to a different name.
+       * `namespace`: `sample-domain1-ns`
+       * `weblogicCredentialsSecretName`: `sample-domain1-weblogic-credentials`
+       * `domainHomeImageBase`: `container-registry.oracle.com/middleware/weblogic:12.2.1.3`
```

Comment:
Please list `weblogicCredentialsSecretName` ahead of `namespace` in order to match the order in the values file.